apertium-eng.eng.dix

```diff
@@ -48958,6 +48962,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 <e lm="William">         <i>William</i><par n="Mark__np"/></e>
 <e lm="William Tell">    <i>William<b/>Tell</i><par n="Mark__np"/></e>
 <e lm="Williams">        <i>Williams</i><par n="Mark__np"/></e>
+<e lm="Williamson">      <i>Williamson</i><par n="Mark__np"/></e>
```

Comment:
This is a personal name?

Reply:
sorry he is cricket player. and its a last name. its a fault i will delete it. :(